code/modules/jobs/job_types/cook.dm

```diff
@@ -88,6 +88,17 @@
 		if(!visualsOnly)
 			other_chefs.cooks++
 
+/datum/outfit/job/cook/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
```

Comment:
```suggestion
	. = ..()
```

we need to have the `. =` portion here in order to preserve the return value of parent (in case it's needed in the future). other instances of `post_equip()` do this in my cursory search. you can read more about this here https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#exception---

Reply:
Thanks for referring me to that doc, I first touched tg code like a year ago and honestly I forgot it existed. That clarifies pretty much all the things I was still unsure about regarding byond.